src/App.js

```diff
@@ -14,11 +14,59 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: ['Jam'],
+  }
+
+  render() {
+    const { goods } = this.state;
+
+    return (
+      <div className="App">
+        <div className="top">
+          <h1>
+            {goods.length === 0
+              ? 'No goods selected'
+              : (`${goods.map(good => (` ${good}`))} selected`)}
+          </h1>
+          {goods.length === 0
+            ? ''
+            : (
+              <button
```

Comment:
```suggestion
          {!!goods.length && (
            <button
```
Let's use conditional rendering like this